install_config/persistent_storage/topics/glusterfs_example_basic_external.adoc

```diff
@@ -0,0 +1,43 @@
+. In your inventory file, add `glusterfs` in the `[OSEv3:children]` section to
+enable the `[glusterfs]` group:
++
+----
+[OSEv3:children]
+masters
+nodes
+glusterfs
+----
+
+. (Optional) Include any of the following role variables in the `[OSEv3:vars]`
+section you wish to change:
++
+----
+[OSEv3:vars]
+openshift_storage_glusterfs_namespace=glusterfs <1>
+openshift_storage_glusterfs_name=storage <2>
```

Comment:
I think there are more options necessary here if this is for Container-Ready storage
openshift_storage_glusterfs_is_native=false    
openshift_storage_glusterfs_storageclass=true
openshift_storage_glusterfs_heketi_is_native=true
openshift_storage_glusterfs_heketi_executor=ssh
openshift_storage_glusterfs_heketi_ssh_port=22
openshift_storage_glusterfs_heketi_ssh_user=root
openshift_storage_glusterfs_heketi_ssh_sudo=false
openshift_storage_glusterfs_heketi_ssh_keyfile="/root/.ssh/id_rsa"   
openshift_storage_glusterfs_block_deploy=false